src/sentry/static/sentry/app/views/organizationIncidents/details/relatedIssues/issuesFetcher.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class IssuesFetcher extends React.PureComponent {
+  static propTypes = {
+    api: PropTypes.object,
+    issueIds: PropTypes.arrayOf(PropTypes.string),
+  };
+
+  state = {
+    loading: true,
+    issues: null,
+    error: null,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.issueIds !== this.props.issueIds) {
+      this.fetchData();
+    }
+  }
+
+  fetchData = async () => {
+    const {api, issueIds} = this.props;
+    this.setState({loading: true});
+
+    if (!issueIds) {
```

Comment:
If there are no issues, how does this component leave its loading state?

Reply:
Oops, thanks

Reply:
Fixed by moving where we set loading state: https://github.com/getsentry/sentry/pull/13617/files#diff-488f70254ef5984b677ed6927ad48ca9R33